### ISSUE TITLE
[help simplify]: GitOps

### ISSUE THREAD

**Issue by CathPag** [cncf/glossary#372]
### Details (optional)

Hello, this [GitOps definition](https://github.com/cncf/glossary/blob/main/content/en/gitops.md) needs to be simplified (e.g. less technical terms). Please refer to the [style guide](https://glossary.cncf.io/style-guide/) before getting started.

**Comment by akshatcoder-hash** [comment]
I would love to simplify this term. Can you please assign it to me? @CathPag

**Comment by CathPag** [comment]
@akshatcoder-hash, @hitarth01 asked for this 20 days ago. I confirmed but forgot to assign the issue.  
@hitarth01, are you still interested in working on this? If not, I'll assign it to @akshatcoder-hash

**Comment by hitarth01** [comment]
@CathPag yes, you can definitely assign this to @akshatcoder-hash.

**Comment by CathPag** [comment]
Thanks for confirming, @hitarth01! @akshatcoder-hash it's all yours. Please note that we already have a [GitOps definition](https://github.com/cncf/glossary/blob/main/content/en/gitops.md) that needs to be simplified (it's still too technical for this Glossary). Also, make sure to read the [style guide](https://glossary.cncf.io/style-guide/) before getting started.

**Comment by akshatcoder-hash** [comment]
Hey @CathPag , please have a look here.
![image](https://user-images.githubusercontent.com/69577224/157090247-ccd97737-8d45-4038-982f-9c47283dfc73.png)

**Comment by Adi9235** [comment]
Hey can I work on this issue @CathPag

**Comment by CathPag** [comment]
Sure! You picked a challenging one! :) 
I'm sure you saw it, but a GitOps definition already exists (linked above). We are trying to simplify that not necessarily start a new one from scratch.

**Comment by Adi9235** [comment]
> Sure! You picked a challenging one! :) I'm sure you saw it, but a GitOps definition already exists (linked above). We are trying to simplify that not necessarily start a new one from scratch.

Yeah, I saw it under open issues and I also wanted to ask that this is my first issue under CNCF so in how much time should I complete this task? @CathPag

**Comment by CathPag** [comment]
We've had a few GitOps experts working on the first version — it's a tough one. So take your time. As long as you are actively working on it, it's fine. But we don't want people to sit on them without doing any work, preventing others from tackling it. Does that make sense? Also, please carefully review the contributor and style guides and reach out on the #glossary channel if you have any questions.

**Comment by Adi9235** [comment]
> We've had a few GitOps experts working on the first version — it's a tough one. So take your time. As long as you are actively working on it, it's fine. But we don't want people to sit on them without doing any work, preventing others from tackling it. Does that make sense? Also, please carefully review the contributor and style guides and reach out on the #glossary channel if you have any questions.

Okay I will try to put my best efforts in this task

**Comment by MaryamTavakkoli** [comment]
I can work on this @CathPag

**Comment by CathPag** [comment]
Here you go :)

**Comment by nate-double-u** [comment]
Updated in: https://github.com/cncf/glossary/pull/2121
